examples/command_error.py

```diff
@@ -0,0 +1,71 @@
+from discord import Intents, Thread, DiscordException
+from discord.ext.commands import (
+    Context,
+    Bot,
+    CheckFailure,
+    BotMissingPermissions,
+    guild_only,
+    NoPrivateMessage,
+    bot_has_permissions,
+    check,
+    when_mentioned_or
+)
```

Comment:
We don't need this many imports, just `import discord` and `from discord.ext import commands` is sufficient and maintains consistency with the other examples.